src/components/ConfigScreen/ConfigScreen.tsx

```diff
@@ -1,25 +1,25 @@
-import { Component, ChangeEvent } from 'react';
-import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
 import {
-  Heading,
+  Button,
+  CheckboxField,
   Form,
-  Workbench,
-  Paragraph,
-  TextField,
-  Notification,
+  Heading,
   Icon,
-  Button,
-  TextLink,
   List,
   ListItem,
-  CheckboxField,
+  Notification,
+  Paragraph,
   Subheading,
+  TextField,
+  TextLink,
+  Workbench,
 } from '@contentful/forma-36-react-components';
+import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
 import ImgixAPI, { APIError } from 'imgix-management-js';
 import debounce from 'lodash.debounce';
+import { ChangeEvent, Component } from 'react';
 
-import './ConfigScreen.css';
 import packageJson from './../../../package.json';
+import './ConfigScreen.css';
 
 export interface AppInstallationParameters {
   imgixAPIKey?: string;
```

Comment:
Ugh, this is prettier apols for the busy dif 🤦🏼

Reply:
Would it be too much trouble to back track all of the prettier changes and open a separate PR to sync everything? Feels like it's added quite a bit of bloat to this changeset.

Reply:
Done ✅